### DISCUSSION TITLE
Extract simple table, but problem

### DISCUSSION THREAD

**Discussion by skornilov30** [jsvine/pdfplumber#772] [Ask for help with specific PDFs]
Hi, try to extract simple table from PDF file.
My debug code:
```
import pdfplumber
## import ghostscript
PDF = r'C:\Users\Sergey\PycharmProjects\pythonProject\pdf\test.pdf'
pdf = pdfplumber.open(PDF)
p16 = pdf.pages[0]
im = p16.to_image()
table_settings = {
    "vertical_strategy": "lines",
    "horizontal_strategy": "lines",
}
im.debug_tablefinder(table_settings)
im.save(r'C:\Users\Sergey\PycharmProjects\pythonProject\pdf\1.bmp', 'bmp')
```
My PDF file:
[test.pdf](https://github.com/jsvine/pdfplumber/files/10152683/test.pdf)

My debug result:
![debug](https://user-images.githubusercontent.com/102043954/205600731-b35232dd-35f3-46e0-ade2-4a55510f3de6.jpg)

Table cells are additionally split, this does not allow me to read the whole table normally as it is drawn in the PDF file. 
What am I doing wrong?

**Comment by skornilov30** [comment]
I found the solution in Discussions. Solved. Thank's!!!

**Reply by codePigPig** [reply]
May I ask which discussion you are referring to? I would like to know, thank you.
